lib/kino/bridge.ex

```diff
@@ -257,10 +258,11 @@ defmodule Kino.Bridge do
 
     result =
       receive do
-        {:io_reply, ^ref, {:error, {:request, _}}} -> {:error, :unsupported}
-        {:io_reply, ^ref, {:error, :request}} -> {:error, :unsupported}
+        {:io_reply, ^ref, {:error, {:request, _}}} -> {:request_error, :unsupported}
+        {:io_reply, ^ref, {:error, :request}} -> {:request_error, :unsupported}
+        {:io_reply, ^ref, {:error, :terminated}} -> {:request_error, :terminated}
```

Comment:
With this we don't actually need to change anything on Livebook side.

In retrospect, we should probably have a nested ok/error tuple to more clearly distinguish communication errors from expected errors, but it's fine.